README.md

```diff
@@ -1,5 +1,9 @@
 # synthetic-publication
 
+[![Coverage Status](https://coveralls.io/repos/github/Financial-Times/coco-synthetic-image-publication-monitor/badge.svg?branch=master)](https://coveralls.io/github/Financial-Times/coco-synthetic-image-publication-monitor?branch=master)
```

Comment:
It is good that we have a coverage badge here, so now we can see that the service has 2% coverage which is something that should be improved in a separate task (not necessarily as part of OpsCop work).